src/MessageQueue/MessageQueue.php

```diff
@@ -68,12 +71,19 @@ public function __construct(string $url, SqsClient $sqsClient)
      */
     public function push(MessageInterface $message)
     {
+        if ($message instanceof DelayedMessageInterface && $message->getDelay() > (15 * 60)) {
+            throw new RuntimeException(sprintf(
+                'SQS only support delayed message for up to 900 seconds (15 minutes), "%s" given',
```

Comment:
That's how it was done but doing it here allows to validate even if people are not using directly the default DelayedMessage (we're doing that actually in our own event where we only implement the interface).